.pre-commit-config.yaml

```
@@ -73,12 +73,12 @@ repos:
   #      - id: mypy
 
   - repo: https://github.com/pre-commit/mirrors-prettier
-    rev: v3.0.0-alpha.2
+    rev: v3.0.0-alpha.3
     hooks:
       - id: prettier
         args: [--prose-wrap=always, --print-width=88]
   - repo: https://github.com/DanielNoord/pydocstringformatter
-    rev: 93b15ca # TODO: Change this when v8.0 is released
+    rev: v0.7.2 # TODO: Change this when v8.0 is released
```

Revert unwanted update
```suggestion
    rev: 93b15ca # TODO: Change this when v8.0 is released
```